sdk/core/azure-core/src/context.cpp

```diff
@@ -8,8 +8,8 @@ using time_point = std::chrono::system_clock::time_point;
 
 Context& Azure::Core::Context::GetApplicationContext()
 {
-  static Context ctx;
-  return ctx;
+  static Context context;
```

Comment:
Make some source code change modifications to trigger GenerateRelease CI steps (which is the one that has link verification).

@sima-zhu, @weshaggard - README updates like the following PR (https://github.com/Azure/azure-sdk-for-cpp/pull/1921) don't trigger the link verification step in CI means we end up checking-in "bad" links and other, and then unrelated PRs then start failing (because they are the ones actually running that step). Can we fix this so any markdown updates run the link verification step (while keeping some of the CI optimization where we don't run other build/test legs)?

cc @danieljurek